package.json

```diff
@@ -70,7 +70,8 @@
     "np": "^7.2.0",
     "preact-x": "npm:preact@10.3.1",
     "prettier": "^2.0.5",
-    "react": "16.13.x",
+    "react": "17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
`react-dom` is needed in order to run unit test somehow? unsure why this is needed

Reply:
Yep I think it's needed by react-testing-library IIRC.